### DISCUSSION TITLE
Synaesthesia scores not being calculated correctly

### DISCUSSION THREAD

**Discussion by PapaNoJohns** [ppy/osu#24549] [Q&A] [answered]
![image](https://github.com/ppy/osu/assets/137879907/581e5c2d-1506-4a96-bb92-7f66ed7ecb50)

Scores using Synesthesia mod are stuck in "still being calculated" state and not updating my total pp on my profile. 

https://lazer.ppy.sh/users/32695872

**Comment by ItsShamed** [comment]


**Reply by peppy** [reply]
Is it? Can you link to documentation of this??

**Reply by ItsShamed** [reply, marked as answer]
#23396 I think? Unless things have changed since.

**Reply by peppy** [reply]
Thanks.
